.github/workflows/appimage.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - arch: [x64, qemu-host]
+          - arch: [ubuntu-latest]
```

Comment:
James I moved to lesser runners where it seemed possible. Let me know if you have any issues with these or if we should adjust these as we get them in.

Reply:
I think it's fine, as long as we're okay with the new speed(s) that result. We'll know very soon if it's a problem or not otherwise.